mnefun/_sss.py

```diff
@@ -779,9 +785,9 @@ def _head_pos_annot(p, subj, raw_fname, prefix='  '):
     if p is not None and p.movecomp is None:
         head_pos = fit_data = None
     else:
-        t_window = _get_t_window(p, raw)
+        # t_window = _get_t_window(p, raw)    SMB 20.08.20: incorp. below
```

Comment:
you can just delete this line, it will live on in `git` history/blame if we ever need it again. Nice that these changes make the code more DRY !

Reply:
The one extra fix I made here, just for accounting purposes, is to set `t_window` to _None_ along with the other parameters.  It's not required unless the subsequent `if` clause is satisfied, but I believe it's good practice if an early `return` is not explicitly declared.